### ISSUE TITLE
gan-loss

### ISSUE THREAD

**Issue by liangzifei** [deepak112/Keras-SRGAN#32]
Dear, 
I saw three values in printing gan-loss:
[0.020138385, 0.018311221, 1.8271642]

Could you please tell me what loss they are represented ?

Best

**Comment by BassantTolba1234** [comment]
please , did you get the answer ?

**Comment by liangzifei** [comment]
The first is the sum loss A+0.001*B, second is generate loss, third is discriminator loss.

**Comment by BassantTolba1234** [comment]
Please all,
I need the code of implementation this part 
the part is 

{The SRResNet networks
were trained with a learning rate of 10−4 and 106 update
iterations. We employed the trained MSE-based SRResNet
network as initialization for the generator when training
the actual GAN to avoid undesired local optima.{

**Comment by neilthefrobot** [comment]
The first loss returned from train_on_batch when there are multiple losses is always a sum of all the losses.
The next number is the VGG perceptual loss (how close the output is to the target output in VGG space)
The last number is the GAN loss (how well the generated images are fooling the discriminator)
